CADETProcess/processModel/componentSystem.py

```diff
@@ -182,7 +182,7 @@ class ComponentSystem(Structure):
     name = String()
 
     def __init__(
-            self, components=None, name=None, charges=None, molecular_weights=None):
+            self, components=None, name=None, charges=None, molecular_weight=None):
```

Comment:
Let's be consistent and use plural (`molecular_weight*s*`) for the `ComponentSystem` and singular for the Component / `Species`